_timeline/2016-05-18-Hardware-prototype-validation.md

```diff
@@ -0,0 +1,5 @@
+---
+title: Hardware Prototype Validation
+date:  2016-05-18
+---
+We tested meshing indoors with our first [hardware prototype](https://github.com/tomeshnet/prototype-cjdns-pi2), it's a success!
```

Comment:
`s/hardware prototype/Raspberry Pi-based prototype` `s/it's a success/covering multiple floors of the Bahen Centre at the University of Toronto`